### PR TITLE
Mitigate #60154

### LINKED ISSUE #60154: Test failure: JIT/HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.sh Timed Out
Run: [runtime-coreclr gcstress0x3-gcstress0xc 20211003.1](https://dev.azure.com/dnceng/public/_build/results?buildId=1400353&view=ms.vss-test-web.build-test-results-tab&runId=40669192&paneView=debug&resultId=111142)
Failed test;
```
CoreCLR Linux arm64 Checked gcstress0x3 @ (Ubuntu.1804.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-20210531091519-97d8652

- JIT/HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.sh

CoreCLR Linux x64 Checked gcstress0x3 @ Ubuntu.1804.Amd64.Open

- readytorun/coreroot_determinism/coreroot_determinism/coreroot_determinism.sh
- JIT/HardwareIntrinsics/General/Vector128/Vector128_ro/Vector128_ro.sh
- JIT/HardwareIntrinsics/*

CoreCLR OSX arm64 Checked gcstress0x3 @ OSX.1100.ARM64.Open

- JIT/HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.sh
- JIT/HardwareIntrinsics/General/Vector128/Vector128_ro/Vector128_ro.sh

CoreCLR Linux arm Checked gcstress0x3 @ (Ubuntu.1804.Arm32.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-bfcd90a-20200121150440

- JIT/HardwareIntrinsics/General/Vector64/Vector64_ro/Vector64_ro.sh
- JIT/HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.sh
- JIT/HardwareIntrinsics/General/Vector128/Vector128_ro/Vector128_ro.sh

CoreCLR windows arm64 Checked gcstress0x3 @ Windows.10.Arm64v8.Open

- JIT\\HardwareIntrinsics\\General\\Vector256\\Vector256_ro\\Vector256_ro.cmd

CoreCLR Linux arm Checked gcstress0xc @ (Ubuntu.1804.Arm32.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-bfcd90a-20200121150440

- JIT/HardwareIntrinsics/General/Vector128/Vector128_r/Vector128_r.sh
- JIT/HardwareIntrinsics/General/Vector256/Vector256_r/Vector256_r.sh
```
**One of error message:**
```
cmdLine:/root/helix/work/workitem/e/JIT/HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.sh Timed Out (timeout in milliseconds: 3600000 from variable __TestTimeout, start: 10/3/2021 2:59:49 PM, end: 10/3/2021 3:59:49 PM)

Return code:      -100
Raw output file:      /root/helix/work/workitem/uploads/Reports/JIT.HardwareIntrinsics/General/Vector256/Vector256_ro/Vector256_ro.output.txt
Raw output:
BEGIN EXECUTION
/root/helix/work/correlation/corerun -p System.Reflection.Metadata.MetadataUpdater.IsSupported=false Vector256_ro.dll ''
Beginning test case Abs.Byte at 10/3/2021 2:59:53 PM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 10/3/2021 3:00:15 PM
Beginning test case Abs.Double at 10/3/2021 3:00:15 PM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 10/3/2021 3:00:23 PM
Beginning test case Abs.Int16 at 10/3/2021 3:00:23 PM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 10/3/2021 3:00:27 PM
Beginning test case Abs.Int32 at 10/3/2021 3:00:27 PM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 10/3/2021 3:00:33 PM
Beginning test case Abs.Int64 at 10/3/2021 3:00:33 PM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 10/3/2021 3:00:39 PM
Beginning test case Abs.SByte at 10/3/2021 3:00:39 PM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenario_UnsafeRead
Beginning scenario: RunClsVarScenario
Beginning scenario: RunLclVarScenario_UnsafeRead
Beginning scenario: RunClassLclFldScenario
Beginning scenario: RunClassFldScenario
Beginning scenario: RunStructLclFldScenario
Beginning scenario: RunStructFldScenario

Ending test case at 10/3/2021 3:00:44 PM
Beginning test case Abs.Single at 10/3/2021 3:00:44 PM
Random seed: 20010415; set environment variable CORECLR_SEED to this value to repro

Beginning scenario: RunBasicScenario_UnsafeRead
Beginning scenario: RunReflectionScenar


Stack trace
   at JIT_HardwareIntrinsics._General_Vector256_Vector256_ro_Vector256_ro_._General_Vector256_Vector256_ro_Vector256_ro_sh()
```

|Build|Kind|Start Time|
|---|---|---|
[1582767](https://dev.azure.com/dnceng/public/_build/results?buildId=1582767)|Rolling|2022-31-01|
[1583845](https://dev.azure.com/dnceng/public/_build/results?buildId=1583845)|Rolling|2022-01-02|
[1585006](https://dev.azure.com/dnceng/public/_build/results?buildId=1585006)|Rolling|2022-01-02|
[1589146](https://dev.azure.com/dnceng/public/_build/results?buildId=1589146)|Rolling|2022-03-02|

### PR DESCRIPTION
**Pull request by echesakov**
Mitigate #60154 by disabling intrinsics that throw `NotSupportedPlatformException` on the corresponding platforms

### CODE REVIEW

File: src/tests/issues.targets [hunk 1]
```diff
@@ -568,6 +605,16 @@
...
             <Issue>Needs Triage</Issue>
         </ExcludeList>
+        <!-- X64 does not support Vector64 -->
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/HardwareIntrinsics/General/Vector64/**">
+            <Issue>https://github.com/dotnet/runtime/issues/60154</Issue>
+        </ExcludeList>
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/HardwareIntrinsics/General/Vector64_1/**">
+            <Issue>https://github.com/dotnet/runtime/issues/60154</Issue>
+        </ExcludeList>
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/HardwareIntrinsics/Arm/**">
```
Comment (sbomer) [inline src/tests/issues.targets:615]: nit: Add a comment like the above (X64 does not support Arm hardware intrinsics)

**Review by sbomer — APPROVED** [review]
Thank you!

**Review by tannergooding — APPROVED** [review]
Just calling out that the `IsSupported == false` scenario will be covered in the outerloop tests where we toggle the `COMPlus_Enable...=0` switches

**Comment by echesakovMSFT** [comment]
Looked at the Tests tab in Azure DevOps and it seems that the change does what was intended.

The failures with macOS Helix queues are irrelevant and fixed in #64565

**Pull request merged**